docs/navigation-configuration.md

```diff
@@ -1,242 +1,242 @@
 # Navigation configuration
 
-Navigation parameters allow you to specify routing configuration, set the appearance of navigation, and define navigation structure.
+Read these guides to get started with configuring your navigation:
+
+* [Navigation elements](#navigation-elements) 
+* [First steps](#first-steps)
+* [Basic navigation parameters](#basic-navigation-parameters)
+* [Grouping navigation nodes](#grouping-navigation-nodes)
+  * [category](#category)
+  * [viewGroup](#viewGroup)
+* [Creating a dynamic path](#creating-a-dynamic-path)
+  * [pathSegment variables](#pathsegment-variables)
+  * [viewUrl variables](#viewurl-variables)
+  * [Node variables](#node-variables)
+  * [Dynamic viewUrl](#dynamic-viewurl)
+
+If you are already familiar with the basics, take a look at:
+* [Full reference list of navigation parameters](navigation-parameters-reference.md)
 
 ## Navigation elements
 
-The image shows the elements of Luigi navigation: 
+There are three main elements to Luigi: 
 
 1. Top navigation which displays the main navigation path.
 2. Side navigation which displays the defined applications.
 3. Main content window which renders the micro frontend.
  
 
-![Navigation layout](assets/navigation-structure.png)
+![Navigation layout](https://github.com/SAP/luigi/blob/master/docs/assets/navigation-structure.png)
 
 
-## Navigation structure
+## First steps
 
-The navigation structure is a recursive tree-like data structure that defines all possible navigation paths within the application. 
+Find the `basicConfiguration.js` file in your luigi-config folder. You can configure the navigation by editing this file. 
 
->**NOTE:** This document describes the navigation structure along with the options you can use to create it. For a full list of available parameters, see the [parameter reference](navigation-parameters-reference.md) document.
+You will notice that the file consists of a tree-like structure of **navigation nodes**. The outer nodes represent the top navigation, while their children represent the side navigation. The nodes have some basic properties (called **navigation parameters**) such as labels, links, views, and (optionally) children.
```

Comment:
I am not sure if it is clear what 'outer nodes' means. Maybe we can say 'first level nodes' or something similar.

Reply:
Thank you for the thorough review, I really appreciate it! 🙇‍♀